app/models/concerns/purgeable.rb

```diff
@@ -23,13 +24,14 @@ def table_key
     end
 
     def fastly
-      return if Rails.env.development?
+      return false if Rails.env.development? ||
+        !(ApplicationConfig["FASTLY_API_KEY"].present? || ApplicationConfig["FASTLY_SERVICE_ID"].present?)
```

Comment:
This might be a little easier to read if you broke it up into 2 lines and used `.blank?` instead of `!(.present?)` 
```ruby
return false if Rails.env.development?
return false if ApplicationConfig["FASTLY_API_KEY"].blank? || ApplicationConfig["FASTLY_SERVICE_ID"].blank?
```